packages/xrpl/test/models/XChainCreateClaimID.ts

```diff
@@ -0,0 +1,84 @@
+import { assert } from 'chai'
+import { validate, ValidationError } from 'xrpl-local'
+import { validateXChainCreateClaimID } from 'xrpl-local/models/transactions/XChainCreateClaimID'
+
+/**
+ * XChainCreateClaimID Transaction Verification Testing.
+ *
+ * Providing runtime verification testing for each specific transaction type.
+ */
+describe('XChainCreateClaimID', function () {
+  let tx
+
+  beforeEach(function () {
+    tx = {
+      Account: 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh',
+      XChainBridge: {
+        LockingChainDoor: 'rGzx83BVoqTYbGn7tiVAnFw7cbxjin13jL',
+        LockingChainIssue: 'XRP',
+        IssuingChainDoor: 'r3kmLJN5D28dHuH8vZNUZpMC43pEHpaocV',
+        IssuingChainIssue: 'XRP',
```

Comment:
Should there be a test for if any of these "Locking" fields are missing in the XChainBridge?